src/test/java/seedu/address/logic/commands/ExportCommandTest.java

```diff
@@ -74,13 +74,6 @@ public void execute_unableToCreateCsvDirectory_throwsCommandException() throws E
         assertEquals("Could not create directory for CSV file.", thrown.getMessage());
     }
 
-    @Test
-    public void execute_withoutValidJsonFilePath_throwsCommandException() {
-        CommandException thrown = assertThrows(CommandException.class, () -> exportCommand
-                .execute(modelWithNoFilePath));
-        assertEquals("Cannot find JSON file to export.", thrown.getMessage());
-    }
-
```

Comment:
good catch! this case is not handled by export class and so we don't need to test it.